src/server/game/Entities/Creature/Creature.cpp

```diff
@@ -1457,6 +1460,7 @@ void Creature::SaveToDB(uint32 mapid, uint8 spawnMask, uint32 phaseMask)
     }
 
     data.spawntimesecs = m_respawnDelay;
+    data.spawntimesecs = m_baseRespawnDelay;
```

Comment:
Why is this being set twice?

Reply:
As I originally understood, Two different fields. One is stock, one is altered by the dynamic respawn system. Saved in two spots in the DB, one as a base control, and the other gets modified. 

I’ll double check but I believe it needs to be like this for the system, thanks for pointing it out.

Reply:
They both change the same variable though, both are data.spawntimesecs. If they were different it would make sense.